admin/class-admin-apple-json.php

```diff
@@ -317,9 +342,12 @@ private function save_json() {
 			return;
 		}
 
-		// Get the specs for the component and theme.
+		// Get the selected subcomponent.
+		$subcomponent = $this->get_selected_subcomponent();
+
+		// Get the specs for the component or subcomponent.
 		$theme = sanitize_text_field( wp_unslash( $_POST['apple_news_theme'] ) );
-		$specs = $this->get_specs( $component, $theme );
```

Comment:
This was a goof - previously, `get_specs` only accepted one argument, and a second argument was being passed here erroneously. We've added an actual second argument to `get_specs` as part of this PR, so `$subcomponent` is being passed as the second parameter below.